packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -621,8 +621,10 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
    */
   private disableTextInput() {
     const inputControl = this.input.shadowRoot.querySelector<HTMLInputElement>('input');
-    inputControl?.setAttribute('disabled', 'true');
-    inputControl.value = '';
+    if (inputControl) {
+      inputControl?.setAttribute('disabled', 'true');
```

Comment:
because there's a if wrapping these lines the conditional access is no longer needed